### PR TITLE
Update Add.java

### PR DESCRIPTION
**Pull request by Bharatbushan2526**
Updated code

### CODE REVIEW

File: Calculator.java [hunk 1]
```diff
@@ -0,0 +1,12 @@
+class Calculator {
+
+	public static void main(String[] args){
+	System.out.println("hello world");
```
Comment (rajatjindal) [inline Calculator.java]: Code is not intended correctly. Format code before checkin

**Review by rajatjindal — CHANGES_REQUESTED** [review]

File: Calculator.java (hunk 1 above)
Comment (Bharatbushan2526) [reply Calculator.java]: Hi sir, I have formatted the code now , pls check it

**Comment by Bharatbushan2526** [comment]
Sir  I have formatted the code now , pls check it so I can pull remaining
3requests

**Review by rajatjindal — APPROVED** [review]

**Comment by rajatjindal** [comment]
congrats on your first Pull Request

**Pull request merged**